### PR TITLE
added entries for youte and tweetexploR

### PR DESCRIPTION
**Pull request by boyd-nguyen**

### CODE REVIEW

**Comment by Quiet27** [comment]
@boyd-nguyen thanks for requesting the review. I should be able to get this done by Thursday 19/01/23, is that OK or do you need it sooner?

**Comment by boyd-nguyen** [comment]
@Quiet27 No problem at all! It's not urgent, I just thought I'd add the entries in case I forget

File: Australian Digital Observatory/content/resources/tweetexplor/contents.lr [hunk 1]
```diff
@@ -0,0 +1,17 @@
+title: tweetexploR
+---
+access: Open sourced under MIT licence
+---
+category: software
+---
+datePublished: 2022-09-07
+---
+description: tweetexploR is an R package for exploring and visualising a collection of Tweets that has been tidied into an SQLite database file by the tidy_tweet library.
```
Comment (Quiet27) [inline Australian Digital Observatory/content/resources/tweetexplor/contents.lr:9]:
```suggestion
description: tweetexploR is an R package for exploring and visualising a collection of Tweets that has been tidied into an SQLite database file by the <a href="https://pypi.org/project/tidy-tweet/" target="_blank">tidy_tweet</a> library.
```

**Review by Quiet27 — APPROVED** [review]
Looks good to me 😺 I just added one suggestion to include a link to tidy_tweet.

**Review by Quiet27 — APPROVED** [review]

**Comment by boyd-nguyen** [comment]
Thanks @Quiet27 Alice for reviewing this and for your suggested change!

**Pull request merged**